src/main/java/uk/gov/hmcts/reform/pip/subscription/management/models/SubscriptionsSummaryDetails.java

```diff
@@ -1,20 +1,18 @@
 package uk.gov.hmcts.reform.pip.subscription.management.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
+import lombok.ToString;
 
 import java.util.ArrayList;
 import java.util.List;
 
 @Data
 public class SubscriptionsSummaryDetails {
-    @JsonProperty("CASE_URN")
+    @ToString.Include(name = "CASE_URN")
     private List<String> caseUrn = new ArrayList<>();
```

Comment:
This model needs the JsonProperty annotation rather than the ToString

Reply:
awesome thanks!